### ISSUE TITLE
Many PilotToolTip "Can't create an ImageOutputStream!" errors logs per turn

### ISSUE THREAD

**Issue by ObviousTech** [MegaMek/megamek#6003]
### Prerequisites and Pre-Issue Checklist

- [X] I'm reporting the issue to the correct repository:
- [MegaMek](https://github.com/MegaMek/megamek/issues)
- [MegaMekLab](https://github.com/MegaMek/megameklab/issues)
- [MekHQ](https://github.com/MegaMek/mekhq/issues)

- [X] I've tested the issue against at least the latest MILESTONE version
- [X] I've asked on the MegaMek Discord about the error
- [X] I've reviewed the BattleTech rules and MegaMek documentation, and I've confirmed that something isn't working as intended.
- [X] I've searched the Github tracker and haven't found the issue listed

### Severity *

Low (Minor/Nuisance): Minor glitches or cosmetic issues that don’t affect gameplay and occur rarely.

### Brief Description *

Nothing seems to be working incorrectly in game from what I can tell, but there are several of the same type of error being logged per unit's turn.    In a 12 round game with about 20 units on each side, this resulted in over 6000 of this error in the megamek log.

The same errors were also logged to megamek.log on a remote client that was connected to the host player.

Error is like 

```
ERROR [megamek.client.ui.swing.tooltip.PilotToolTip] {AWT-EventQueue-0}
megamek.client.ui.swing.tooltip.PilotToolTip.crewPortraits(PilotToolTip.java:221) - 
javax.imageio.IIOException: Can't create an ImageOutputStream!
... (+42 more lines of error, see attached logs for examples)
```

Some lines of the full error text do vary, depending on the action which led to the error.

### Steps to Reproduce

Steps:
- start a megamek game
- play the game normally.  deploying units, moving, selecting different units, etc.   The same error occurs many times per round.

Actions I checked (with a number corresponding to the number of new errors logged when performing that action):
- placing a unit on the board: 0
- selecting a different unit: 1
- hovering over a unit that is on the board: 0
- going from initiative to movement phase: 4 (when there are 2 units)
- completing a unit's move, followed by the next unit being selected: 3
- moving the last unit, which immediately moves to firing phase: 5
- completing a unit's fire, which selects the next unit: 2
- completing the last unit's fire, ending the firing phase: 1
- fire results to end phase: 1

other variables tested that did not seem to affect the whether the error occurs:
- pilot having a portait vs no portrait
- camo selected vs no camo

### Operating System *

Windows 10

### Java Version *

Java Version: 17.0.12

### MegaMek Suite Version *

v0.50.0

### Attach Files

[megamek.log](https://github.com/user-attachments/files/17005003/megamek.log)

[autosave_2024-09-15_03-35-12.sav.gz](https://github.com/user-attachments/files/17005004/autosave_2024-09-15_03-35-12.sav.gz)

### Final Checklist

- [X] I've checked to make sure that this issue has not already been filed
- [X] I'm reporting only one issue in this ticket for clarity and focus

**Comment by HoneySkull** [comment]
1. Are you using or have setup any custom directories?  Or are you using the defaults?
2. Does the location/folder your megamek suite is installed have write permissions?  
3. Do you have sufficient disk space on the drive you have megamek installed on?
4. Are you using any custom fluf packs or custom graphics that might have invalid dimensions for portraits.

**Comment by ObviousTech** [comment]
> 1. Are you using or have setup any custom directories?  Or are you using the defaults?

not 100% sure what you mean by this / how to do this


> 2. Does the location/folder your megamek suite is installed have write permissions?

i think it should?  the units.cache generated ok.  savegames and logs write ok.  I didn't do anything different from previous versions when setting up.

> 3. Do you have sufficient disk space on the drive you have megamek installed on?

very yes

> 4. Are you using any custom fluff packs or custom graphics that might have invalid dimensions for portraits.

I added the megamek-extras content, plus have custom units+camos from a campaign that uses this install (in the data/ subfolders, not userdata/).   However, none of these were getting used in this example save I posted.

---

I have another mekhq/megamek install of this version which doesn't have the extras added.  Checking if the errors happen there as well... 
Yep, same amount of errors.   Even with just 1 mech on the board, no camo, no portrait selected.  Same `javax.imageio.IIOException: Can't create an ImageOutputStream!` errors.

I turned off the "show pilot portrait in tooltip" option, still get the error.
